packages/runner/src/selector-playground/css.js

```diff
@@ -0,0 +1,166 @@
+export const css = `
```

Comment:
What's the reason this can't remain `scss`?

Reply:
I first wrote the code in this way to understand which code can be reused. 

This code will be completely removed in the final version. I think both `scss` and `css` versions are OK for now.

Reply:
I see. Maybe this PR should also remove [selector-playground.scss](https://github.com/cypress-io/cypress/blob/823ffd0ca920c82170705d9abfa60592b8f3979a/packages/runner/src/selector-playground/selector-playground.scss) and [selector-playground.scss.d.ts](https://github.com/cypress-io/cypress/blob/823ffd0ca920c82170705d9abfa60592b8f3979a/packages/runner/src/selector-playground/selector-playground.scss.d.ts) since it's replacing them?

Reply:
I thought about removing them with the next PR (migrating React `Highlight` component to Vue `Highlight` component). I agree it makes sense to do it now.